tests/manager/manager-executeBulkWrite_error-009.phpt

```diff
@@ -8,41 +8,41 @@ MongoDB\Driver\Manager::executeBulkWrite() with wrong options and values
 require_once __DIR__ . "/../utils/basic.inc";
 
 $manager = new MongoDB\Driver\Manager(REPLICASET);
-    
+
 echo throws(function() use ($manager) {
     $bulk = new MongoDB\Driver\BulkWrite();
-    $bulk->insert(['wc' => 1]);
-    $manager->executeBulkWrite(NS, $bulk, ['writeConcern' => 'foo']);
+    $bulk->insert(['x' => 1]);
+    $manager->executeBulkWrite(NS, $bulk, ['session' => 'foo']);
 }, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
 
 echo throws(function() use ($manager) {
     $bulk = new MongoDB\Driver\BulkWrite();
-    $bulk->insert(['wc' => 1]);
-    $manager->executeBulkWrite(NS, $bulk, ['writeConcern' => 42]);
+    $bulk->insert(['x' => 1]);
+    $manager->executeBulkWrite(NS, $bulk, ['session' =>  new stdClass]);
 }, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
 
 echo throws(function() use ($manager) {
     $bulk = new MongoDB\Driver\BulkWrite();
-    $bulk->insert(['wc' => 1]);
-    $manager->executeBulkWrite(NS, $bulk, ['unknown' => 'foo']);
+    $bulk->insert(['x' => 1]);
+    $manager->executeBulkWrite(NS, $bulk, ['writeConcern' => 'foo']);
 }, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
 
 echo throws(function() use ($manager) {
     $bulk = new MongoDB\Driver\BulkWrite();
-    $bulk->insert(['wc' => 1]);
-    $manager->executeBulkWrite(NS, $bulk, ['readPreference' => 'foo']);
+    $bulk->insert(['x' => 1]);
+    $manager->executeBulkWrite(NS, $bulk, ['writeConcern' => new stdClass]);
 }, 'MongoDB\Driver\Exception\InvalidArgumentException'), "\n";
```

Comment:
You should try to minimize changes to tests. There was no need to change the order, and to change "wc" to "x" for the writeConcern options for example.

Reply:
Noted re: "wc" to "x". The order changed as I basically wrote one test for all options and then copy/pasted it throughout all of the myriad of execute method tests, stripping out irrelevant options for each test based on what the method took.